librz/core/cmd/cmd_print.c

```diff
@@ -1088,31 +1087,31 @@ static void cmd_print_fromage(RzCore *core, const char *input, const ut8 *data,
 	} break;
 	case 'x': // "pFx" x509
 	{
-		RASN1Object *object = rz_asn1_create_object(data, size);
-		RX509Certificate *x509 = rz_x509_parse_certificate(object);
+		RzASN1Object *object = rz_asn1_object_parse(data, size);
+		RzX509Certificate *x509 = rz_x509_certificate_parse(object);
 		if (x509) {
 			RzStrBuf *sb = rz_strbuf_new("");
 			rz_x509_certificate_dump(x509, NULL, sb);
 			char *res = rz_strbuf_drain(sb);
 			if (res) {
-				rz_cons_printf("%s\n", res);
+				rz_cons_printf("%s", res);
 				free(res);
 			}
-			rz_x509_free_certificate(x509);
+			rz_x509_certificate_free(x509);
 		} else {
 			RZ_LOG_ERROR("core: Malformed object: did you supply enough data?\ntry to change the block size (see b?)\n");
 		}
 	} break;
 	case 'p': // "pFp"
 	{
-		RCMS *cms = rz_pkcs7_parse_cms(data, size);
+		RzCMS *cms = rz_pkcs7_cms_parse(data, size);
 		if (cms) {
 			char *res = rz_pkcs7_cms_to_string(cms);
 			if (res) {
-				rz_cons_printf("%s\n", res);
+				rz_cons_printf("%s", res);
```

Comment:
Same question here

Reply:
yes, before it was adding an unnecessary newline